README.md

```diff
@@ -89,6 +90,13 @@ vars:
 
 To connect your multiple schema/database sources to the package models, follow the steps outlined in the [Union Data Defined Sources Configuration](https://github.com/fivetran/dbt_fivetran_utils/tree/releases/v0.4.latest#union_data-source) section of the Fivetran Utils documentation for the union_data macro. This will ensure a proper configuration and correct visualization of connections in the DAG.
 
+#### Disable Country Reports
+This package leverages the `campaign_country_report` table to help report on ad and campaign performance by country. However, if you are not actively syncing this report from your TikTok Ads connection, you may disable the transformations for the `campaign_country_report` by adding the following variable configuration to your root `dbt_project.yml` file:
```

Comment:
```suggestion
This package leverages the `campaign_country_report` table to help report on ad and campaign performance by country and region. However, if you are not actively syncing this report from your TikTok Ads connection, you may disable the transformations for the `campaign_country_report` by adding the following variable configuration to your root `dbt_project.yml` file:

Reply:
See same comment from the source. It's not entirely accurate to claim `and region` here. Instead we'll clarify `country/geographic region level`.